includes/discovery/mempools/fs-switch.inc.php

```diff
@@ -0,0 +1,24 @@
+<?php
+//
+// Discovery for FS-switch devices.
+//
+
+if ($device['os'] == 'fs-switch') {
+    echo 'FS : ';
+    $mempools_array = snmpwalk_cache_oid($device, 'memTotalReal', [], 'SWITCH', 'fs');
+    $mempools_array = snmpwalk_cache_oid($device, 'memTotalFree', $mempools_array, 'SWITCH', 'fs');
+    $mempools_array = snmpwalk_cache_oid($device, 'memTotalUsed', $mempools_array, 'SWITCH', 'fs');
+    d_echo($mempools_array);
+    if (is_array($mempools_array)) {
```

Comment:
is_array is always true, you can just remove the if

Reply:
done